patches/minecraft/net/minecraft/world/gen/layer/BiomeLayer.java.patch

```diff
@@ -0,0 +1,75 @@
+--- a/net/minecraft/world/gen/layer/BiomeLayer.java
++++ b/net/minecraft/world/gen/layer/BiomeLayer.java
+@@ -28,12 +28,32 @@
+    private static final int[] field_202746_t = new int[]{field_202730_d, field_202729_c, field_202741_o, field_202736_j};
```

Comment:
Are these original fields still in use?
If not: Put a  `@Deprecated` in front of them (on the same line) and a comment behind them referencing their replacement (also on the same line)

Eventhough these fields might be private, if vanilla code becomes unused by forge changes. We should keep track of this. Not just for people using the code, but also for porters who port the patch in the future. It is then always good to know what happens with a field.

Do not remove the field since that can also cause issue. As I said a `@Deprecated` and a comment should suffice.

Reply:
Sure, will do